seaborn/categorical.py

```diff
@@ -1498,24 +1498,52 @@ def estimate_statistic(self, estimator, ci, n_boot):
             self.value_label = "{}({})".format(estimator.__name__,
                                                self.value_label)
 
-    def draw_confints(self, ax, at_group, confint, colors, **kws):
-
-        kws.setdefault("lw", mpl.rcParams["lines.linewidth"] * 1.8)
+    def draw_confints(self,
+                      ax, at_group,
+                      confint,
+                      colors,
+                      conf_lw=None,
+                      capsize=None,
+                      **kws):
+
+        if conf_lw:
+            kws.setdefault("lw", mpl.rcParams["lines.linewidth"] * conf_lw)
```

Comment:
This way of doing this is confusing because `conf_lw` sounds like you are specifying a width value, but it's really a scaling factor. I would just set the linewidth directly using this parameter.